scripts/linters/pre_commit_linter.py

```diff
@@ -349,8 +350,7 @@ def _get_all_filepaths(input_path, input_filenames):
             checked, ignored if input_path is specified.
 
     Returns:
-        all_filepaths: list(str). The list of filepaths to be linted and
-        checked.
+        list(str). The list of filepaths to be linted and checked.
```

Comment:
I was fine with the original code too!

Reply:
We are not using `:` in return section anymore.